src/cli.js

```diff
@@ -167,120 +166,97 @@ function checkContributors(argv) {
-              contributor.contributions.push(category)
-            // console.log(
-            //   `Adding ${chalk.blue(usr.login)} for ${chalk.underline(category)}`,
-            // )
-          } //else console.warn(`Oops, I couldn't find any category for the "${lbl}" label`)
-        })
-        const existingContributor = contributorsToAdd.filter(
-          ctrb => ctrb.login === usr.login,
-        )
-        if (existingContributor.length) {
-          existingContributor[0].contributions = [
-            ...new Set(
-              existingContributor[0].contributions.concat(
-                contributor.contributions,
-              ),
-            ),
-          ]
-        } else contributorsToAdd.push(contributor)
-      })
+  reviewers.forEach(usr => {
+    contributorsToAdd.push({login: usr.login, contributions: ['review']})
+
+    console.log(
+      `Adding ${chalk.underline('Reviewer')} ${chalk.blue(usr.login)}`,
+    )
+  })
+
+  issueCreators.forEach(usr => {
+    const contributor = {
+      login: usr.login,
+      contributions: [],
+    }
+
+    usr.labels.forEach(lbl => {
+      const guessedCategory = learner.classify(lbl).find(c => c && c !== 'null')
 
-      repoContributors.commitAuthors.forEach(usr => {
-        // const contributor = {
-        //   login: usr.login,
-        //   contributions: [],
-        // }
-        // console.log('commit auth:', usr)
-        const existingContributor = contributorsToAdd.filter(
-          ctrb => ctrb.login === usr.login,
+      if (!guessedCategory) {
+        console.warn(
+          `Oops, I couldn't find any category for the "${lbl}" label`,
         )
-        if (existingContributor.length) {
-          //there's no label or commit message info so use only code for now
-          if (!existingContributor[0].contributions.includes('code')) {
-            existingContributor[0].contributions.push('code')
-          }
-        } else
-          contributorsToAdd.push({login: usr.login, contributions: ['code']})
-      })
 
-      // console.log('contributorsToAdd=', contributorsToAdd)
-      contributorsToAdd.forEach(contributor => {
+        return
+      }
+
+      if (!contributor.contributions.includes(guessedCategory)) {
+        contributor.contributions.push(guessedCategory)
+
         console.log(
-          `Adding ${chalk.blue(contributor.login)} for ${chalk.underline(
-            contributor.contributions.join('/'),
+          `Adding ${chalk.blue(usr.login)} for ${chalk.underline(
+            guessedCategory,
           )}`,
         )
-        args._ = ['', contributor.login, contributor.contributions.join(',')]
-        // if (contributor.contributions.length) addContribution(args)
-        // else console.log('Skipping', contributor.login)
-      })
-    },
-    err => console.error('fetch error:', err),
-  )
+      }
+    })
+
+    const existingContributor = contributorsToAdd.find(
+      c => c.login === usr.login,
```

Comment:
Do you think `c` would be as clear as a `contributor` or abbreviations like `ctrb`/`ctr`/...?

Reply:
Yea for sure, i was just saving line space. Ill change it up